### PR TITLE
Add player rotation

### PR DESCRIPTION
**Pull request by Nireves333**
Player rotation when arrow keys and WASD keys are pushed, and when using BFS. 
Note that the player model is the robot, its temporary will be changed shortly.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -174,6 +174,10 @@ const renderLevel = (root, levelNumber) => {
     if (soko.goto(row, col)) {
       renderBoard();
     }
+    const playerSequence = soko.sequence()[soko.sequence().length - 1];
```
Comment (ggorlen) [inline src/index.js:177]: Maybe `lastMove` would be clearer--it's not really a `playerSequence`, just a single step.

If `soko.goto()` failed, then are we sure we're not indexing out of bounds here? (I'm just eyeballing here, correct me if I'm wrong)

File: src/index.js [hunk 2]
```diff
@@ -187,23 +191,38 @@ const renderLevel = (root, levelNumber) => {
...
       if (soko.move(moves[event.code])) {
         renderBoard();
-
         if (soko.solved()) {
-          setTimeout(() => { renderLevelComplete(root); }, 100);
           console.log("solved!");
         }
       }
+      const playerEl = document.querySelector(".player, .player-on-goal");
+      playerEl.style.transform = `rotate(${dirToDegree[event.code]}deg)`;
```
Comment (ggorlen) [inline src/index.js:225]: This code is repeated in 2 places and would be a good candidate to be put into its own function, `rotatePlayer`.

**Comment by ggorlen** [comment]
It'd be cool to see the actual sprite we want to use here before merging, otherwise we might need to undo the changes if it turns out that a flat, non-rotating image seems best for the player (I still prefer the idea of a cocktail glass or some other common neon sign to the robots). If the flatness seems weird, keep in mind our "boxes" are also front-facing signs.

File: src/index.js (hunk 1 above)
Comment (itancio) [reply src/index.js]: We can use the str.charAt(length - 1) to get the last character of a string. This method will return an empty string if it's out of bounds.

**Review by itancio — COMMENTED** [review]
Can you add more contrast between the player and player_on_goal - maybe make the one pink skin darker and the other more luminescent and/or change the other hair entirely light blue.

**Comment by ggorlen** [comment]
I'm closing this; there are unforseen UX issues with player rotation (like handling undos properly) and I don't think we're going to have time to resolve them all before the deadline. Let's focus on higher-priority issues, then revisit this if there's time.

**Pull request closed without merging**